### DISCUSSION TITLE
Should we create a framework for using GitHub?

### DISCUSSION THREAD

**Discussion by Schiano-NOAA** [nmfs-ost/asar#31] [Q&A]
Other than the R package itself and creating the set up for quarto, would a good additional feature for the package be something like creating a pre-fabricated GitHub repository by running a function? (If this is even possible)

**Comment by kellijohnson-NOAA** [comment]
GitHub has the ability to store "template" repositories. As well as {usethis} can be helpful for creating files and could be used to help create a function. As I mentioned in our meeting though, I think that this should be placed lower on the priority list because the number one priority is the assessment template and increasing the number of default tables and figures that are present.
